src/playback/view/internal/soundflag/soundflagsettingsmodel.cpp

```diff
@@ -242,7 +244,15 @@ QVariantList SoundFlagSettingsModel::contextMenuModel()
 {
```

Comment:
Ideally, we should not re-build the context menu model, but just update "enabled" of the reset button